prometheus/process_collector_test.go

```diff
@@ -72,4 +78,23 @@ func TestProcessCollector(t *testing.T) {
 			t.Errorf("want body to match %s\n%s", re, buf.String())
 		}
 	}
+
+	brokenProcessCollector := NewProcessCollector(ProcessCollectorOpts{
+		PidFn:        func() (int, error) { return 0, errors.New("boo") },
+		ReportErrors: true,
+	})
+
+	ch := make(chan Metric)
+	go func() {
+		brokenProcessCollector.Collect(ch)
+		close(ch)
+	}()
+	for m := range ch {
+		fmt.Println("feep")
```

Comment:
What is this?

Reply:
Errr… probably just to test the reviewer. :o)

Reply:
Will remove before merging.